### ISSUE TITLE
Data Collection Criteria

### ISSUE THREAD

**Issue by chackoge** [rchampieux/guidelines_citation_patterns#1]
The [eutils](http://www.ncbi.nlm.nih.gov/books/NBK25500/) service will get you a lot of relevant information in XML or json just by submitting a list of pmids. GC

**Comment by rchampieux** [comment]
Yes, Kent is going to play with it when he returns, but I’ll take a look too.  I’m in the air right now – flying to DC, but more soon.
